xds/internal/test/e2e/controlplane.go

```diff
@@ -0,0 +1,63 @@
+/*
+ *
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	xdsinternal "google.golang.org/grpc/internal/xds"
+	"google.golang.org/grpc/xds/internal/testutils/e2e"
+)
+
+type controlPlane struct {
+	server              *e2e.ManagementServer
+	nodeID              string
+	bootstrapContentStr string
```

Comment:
This could be too much of a nit, but I feel the `Str` suffix does not add any value here.

Reply:
Removed